### PR TITLE
Fixes being able to steal AIs out of other intellicards if you're both trying to download it

### LINKED ISSUE #13422: able to download ai from cards
## Test Merges: 
Not Supplied / None
## Reproduction:
If someone starts an AI download, then someone cards the AI, when the first download completes it will force the Ai into the second card, while leaving the first card intact.
 Submitted by: Cowbot (97736956326207488)

### PR DESCRIPTION
**Pull request by Bibby0110**
# Document the changes in your pull request

Fixes #13422

# Wiki Documentation

# Changelog

:cl:  
tweak: Fixes issue where you could end up downloading an AI out of an intellicard
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
Is this tested

File: code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm [hunk 1]
```diff
@@ -205,6 +205,9 @@ GLOBAL_VAR_INIT(ai_control_code, random_nukecode(6))
 
 /obj/machinery/computer/ai_control_console/proc/finish_download()
 	if(intellicard)
+		if(!isaicore(get_turf(downloading)))
```
Comment (adamsong) [inline code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm]:
```suggestion
		if(!isaicore(downloading.loc))
```
Turfs can't  be AI cores

**Review by adamsong — DISMISSED** [review]

File: code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm (hunk 1 above)
Comment (TheGamerdk) [reply code/modules/mob/living/silicon/ai/decentralized/management/ai_controlpanel.dm:208]: get_turf returns loc

**Comment by Bibby0110** [comment]
> Is this tested

why would i do such a thing?

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**